gen-resourcesdocs/README.md

```diff
@@ -0,0 +1,124 @@
+# Kubernetes API resources documentation generator
+
+This tool extracts information from the OpenAPI specification file of the [Kubernetes API](https://github.com/kubernetes/kubernetes/blob/master/api/openapi-spec/swagger.json) and creates documentation in Markdown format, suitable for the [Kubernetes website](https://kubernetes.io/docs/reference/kubernetes-api/).
+
+## Outline
+
+The documentation is splitted in several *parts*, and each part contains several *chapters*, each chapter describing a main Kubernetes resource and its associated resources and definitions, followed by the *Operations* operating on the resources documented in the chapter.
```

Comment:
nit: What do parts and chapters represent?
Possibly reword:
The documentation is split into *parts*. Each part can contain any number of *chapters*. A chapter describes:
- a main Kubernetes resource
- any resources and definitions associated with the main resource
- any *Operations* operating on the resources documented in the chapter

Reply:
fixed, thanks